chapter-12-native-apps/desktop-app/package.json

```diff
@@ -29,7 +29,7 @@
         "@types/node": "^20.11.30",
         "@types/react": "^18.3.2",
         "@types/react-dom": "^18.3.0",
-        "electron": "31.6.0",
+        "electron": "^33.2.0",
```

Comment:
Previously there was a macOS issue that caused tons of console output so I was using an old version. The issue is now fixed so we use latest again.